Library/Homebrew/cmd/install.rb

```diff
@@ -237,6 +237,8 @@ def self.install
                             skip_cask_deps: args.skip_cask_deps?,
                             quarantine:     args.quarantine?,
                             quiet:          args.quiet?).install
+      rescue Cask::CaskAlreadyInstalledError => e
+        opoo e.message
```

Comment:
Previously it worked like this:

https://github.com/Homebrew/brew/blob/cba118533b83ba1587cf65da0142f58523d210e7/Library/Homebrew/cask/cmd/install.rb#L108-L112